src/components/Dropdown.js

```diff
@@ -157,7 +159,7 @@ export default class Dropdown extends React.Component<Props, State> {
 						<DropdownRow text="Profiles" bold />
 						<DropdownRow
 							text={user.username}
-							image={user.avatar ? api.user.avatarPath(user.avatar) : 'gradient'}
+							image={avatar ? api.user.avatarPath(avatar) : 'gradient'}
```

Comment:
Right half of this ternary will never execute since `user.uid` is always present
The goal of this is to display a local gradient image when there is no avatar, because ZEIT’s API for this returns an SVG, which we can’t display

Reply:
I see :( in my case avatar is `null` but using the id works because Zeit has pulled my gravatar directly to the id. So we might have to find an alternative to check the image type before showing it.

Reply:
btw, I think that `react-native-img-cache` is deprecated. We might want to upgrade to https://github.com/DylanVann/react-native-fast-image

Reply:
I don't think we can check image type without fetching it first, in which case we'll have to fetch it twice - for checking the type and via `<Image />`. Even though images are really small, we should avoid extra requests on mobile

Reply:
`react-native-fast-image` looks pretty good, I'll check it out. Thanks for suggesting!

Reply:
> `react-native-fast-image` looks pretty good, I'll check it out. Thanks for suggesting!
No problem!

> I don't think we can check image type without fetching it first, in which case we'll have to fetch it twice - for checking the type and via `<Image />`. Even though images are really small, we should avoid extra requests on mobile
Maybe just one request to detect the type and use the same content to either show an Image or SVG, I know this is not so cool but we don't do it everyone that has the account as I do will get a gray image. You can see an example of this on my other PR.